File: bitget-java-sdk-api/src/main/java/com/bitget/openapi/api/v1/MixMarketApi.java
package com.bitget.openapi.api.v1;

import com.bitget.openapi.dto.response.ResponseResult;
import retrofit2.Call;
import retrofit2.http.GET;
import retrofit2.http.QueryMap;

import java.util.Map;

public interface MixMarketApi {

    @GET("/api/mix/v1/market/contracts")
    Call<ResponseResult> contracts(@QueryMap Map<String, String> paramMap);

    @GET("/api/mix/v1/market/depth")
    Call<ResponseResult> depth(@QueryMap Map<String, String> paramMap);

    @GET("/api/mix/v1/market/ticker")
    Call<ResponseResult> ticker(@QueryMap Map<String, String> paramMap);

    @GET("/api/mix/v1/market/tickers")
    Call<ResponseResult> tickers(@QueryMap Map<String, String> paramMap);

    @GET("/api/mix/v1/market/fills")
    Call<ResponseResult> fills(@QueryMap Map<String, String> paramMap);

    @GET("/api/mix/v1/market/candles")
    Call<ResponseResult> candles(@QueryMap Map<String, String> paramMap);

}

File: bitget-php-sdk-api/src/api/v1/MixMarketApi.php
<?php


namespace bitget\api\v1;


use bitget\internal\BitgetApiClient;

class MixMarketApi
{
    var BitgetApiClient $BitgetApiClient;

    public function __construct($BitgetApiClient)
    {
        $this->BitgetApiClient = $BitgetApiClient;
    }

    public function contracts($params): string
    {
        return $this->BitgetApiClient->doGet("/api/mix/v1/market/contracts", $params);
    }

    public function depth($params): string
    {
        return $this->BitgetApiClient->doGet("/api/mix/v1/market/depth", $params);
    }

    public function ticker($params): string
    {
        return $this->BitgetApiClient->doGet("/api/mix/v1/market/ticker", $params);
    }

    public function tickers($params): string
    {
        return $this->BitgetApiClient->doGet("/api/mix/v1/market/tickers", $params);
    }

    public function fills($params): string
    {
        return $this->BitgetApiClient->doGet("/api/mix/v1/market/fills", $params);
    }

    public function candles($params): string
    {
        return $this->BitgetApiClient->doGet("/api/mix/v1/market/candles", $params);
    }
}